/**:
  ros__parameters:
    question_qty: 8
    state_qty: 8
    max_images_per_state: 10
